modules/azure/api_management_api/variables.tf

```
@@ -223,8 +223,8 @@ variable "custom_backend_policy" {
   default     = null
 }
 
-variable "role_assignment" {
-  type        = string
-  description = "Role to validate in the JWT token's 'roles' claim for access control."
+variable "role_assignments" {
+  type        = list(string)
+  description = "Roles to validate in the JWT token's 'roles' claim for access control."
   default     = null
```

```suggestion
  default     = []
```

I believe the default should be this?

---

yes agree, thx